data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -59,88 +61,88 @@
 # Column subsets
 CENSUS_COUNTIES_COLUMNS = ["USPS", "GEOID", "NAME"]
 TILES_SCORE_COLUMNS = [
-    "GEOID10_TRACT",
-    "State Name",
-    "County Name",
-    "Total population",
-    "Score D (percentile)",
-    "Score D (top 25th percentile)",
-    "Score E (percentile)",
-    "Score E (top 25th percentile)",
-    "Score G (communities)",
-    "Score G",
-    "Definition L (communities)",
-    "Definition L (percentile)",
-    "Poverty (Less than 200% of federal poverty line) (percentile)",
-    "Percent individuals age 25 or over with less than high school degree (percentile)",
-    "Linguistic isolation (percent) (percentile)",
-    "Unemployed civilians (percent) (percentile)",
-    "Housing burden (percent) (percentile)",
-    "Diagnosed diabetes among adults aged >=18 years (percentile)",
-    "Current asthma among adults aged >=18 years (percentile)",
-    "Coronary heart disease among adults aged >=18 years (percentile)",
-    "Life expectancy (years) (percentile)",
-    "Traffic proximity and volume (percentile)",
-    "FEMA Risk Index Expected Annual Loss Score (percentile)",
-    "Energy burden (percentile)",
-    "Wastewater discharge (percentile)",
-    "Percent pre-1960s housing (lead paint indicator) (percentile)",
-    "Diesel particulate matter (percentile)",
-    "Particulate matter (PM2.5) (percentile)",
-    "Median household income (% of AMI) (percentile)",
-    "Percent of individuals < 200% Federal Poverty Line (percentile)",
+    field_names.GEOID_TRACT_FIELD,
+    field_names.STATE_FIELD,
+    field_names.COUNTY_FIELD,
+    field_names.TOTAL_POP_FIELD,
+    field_names.SCORE_D + field_names.PERCENTILE_FIELD_SUFFIX,
```

Comment:
Should we get rid of all the Score D/E/G stuff from the tiles?

Reply:
This was purely to change the file, not do the work of updating everything to have exactly what we want (since that is still TBD and captured in a separate issue).

Reply:
Ok got it!